internal/grpc/services/sharesstorageprovider/sharesstorageprovider_test.go

```diff
@@ -261,7 +261,23 @@ var _ = Describe("Sharesstorageprovider", func() {
 						},
 					}
 				default:
-					if req.Ref.ResourceId.OpaqueId == "shareddir-merged" {
+					switch req.Ref.ResourceId.OpaqueId {
+					case "shareddir", "shareddir2":
```

Comment:
Do we need to provide the constants for the `"shareddir", "shareddir2", "shareddir-merged"`?

Reply:
Looking at the surrounding code, especially the outer switch statement, I'd say: no :smiley: 

More seriously I guess the mocks could benefit from some refactoring, but I think the string constants are the least of on an issue.